docs/package.json

```diff
@@ -79,12 +79,10 @@
     "not dead"
   ],
   "dependencies": {
-    "@types/overlayscrollbars": "^1.12.5",
```

Comment:
Should we include modified `package-lock.json` as well?

Reply:
I'll remove it